runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -159,9 +159,9 @@ There is also some text missing here.
 
  2. At the shell prompt type this command:
 ~~~ sh
-     $ vim tutor
+     $ nvim tutor
 ~~~
-    'vim' is the command to start the Vim editor, 'tutor' is the name of
+    'nvim' is the command to start the Vim editor (it is NeoVim, but we call it as Vim sometimes), 'tutor' is the name of
```

Comment:
"Neovim" not "NeoVim".

Reply:
Actually just change this line to:

    nvim' is the command to start the Nvim editor, 'tutor' is the name of

Reply:
Done